scripts/novaseq/checkfornewrun.bash

```diff
@@ -50,17 +50,29 @@ for RUN_DIR in ${IN_DIR}/*; do
 
     if [[ -f ${RUN_DIR}/RTAComplete.txt ]]; then
         if [[ ! -f ${RUN_DIR}/demuxstarted.txt ]]; then
-            log "date +'%Y%m%d%H%M%S' > ${RUN_DIR}/demuxstarted.txt"
-            date +'%Y%m%d%H%M%S' > ${RUN_DIR}/demuxstarted.txt
+
+            # start with a clean slate: remove empty sample sheets before continuing
+            if [[ ! -s ${RUN_DIR}/SampleSheet.csv && -e ${RUN_DIR}/SampleSheet.csv ]]; then
+                rm ${RUN_DIR}/SampleSheet.csv
+            fi
 
             if [[ ! -e ${RUN_DIR}/SampleSheet.csv ]]; then
                 log "demux sheet fetch --application nova --pad --longest ${FC} > ${RUN_DIR}/SampleSheet.csv"
-                demux sheet fetch --application nova --pad --longest ${FC} > ${RUN_DIR}/SampleSheet.csv
+                demux sheet fetch --application nova --pad --longest ${FC} > ${RUN_DIR}/SampleSheet.csv 
+            fi
+
+            # exit if samplesheet is still empty after running demux sheet fetch
+            if [[ ! -s ${RUN_DIR}/SampleSheet.csv && -e ${RUN_DIR}/SampleSheet.csv ]]; then
```

Comment:
Similar to above. Probably without `-e`.

Reply:
fix in https://github.com/Clinical-Genomics/demultiplexing/pull/103/commits/d1099da8290b8899976efb5aaa7e56eddf228fa8